packages/lightcone_v2.js/src/lib/wallet/ethereum/contracts/AbiFunction.ts

```diff
@@ -1,4 +1,5 @@
-import {methodID, rawDecode, rawEncode} from 'ethereumjs-abi';
+// Using import {*} from 'ethereumjs-abi'; failed to import ethereumjs-abi
+var abi = require('ethereumjs-abi')
```

Comment:
We need to change how to import ethereumjs-abi in TypeScript. Previous method failed to import.